### PR TITLE
BaSP-TG-120: navbar to home

### PR DESCRIPTION
**Pull request by Juanoauil**
### Worked with @dayanagm96 

- Open - Close  navbar in homepage

### CODE REVIEW

File: src/Components/Header/Header.jsx [hunk 1]
```diff
@@ -1,31 +1,84 @@
...
+          <nav className={style.navbar}>
+            {location.pathname !== '/home' ? (
+              <button className={style.closeBtn} onClick={toggleSidebar}>
+                <img src={closeBtn} alt="cross-img" />
+              </button>
+            ) : null}
+            <h2>Menu</h2>
+            <ul className={style.rutes}>
+              <li>
+                <Link className={style.links} to="/timesheets">
```
Comment (cocolobos) [inline src/Components/Header/Header.jsx:26]: remove this Link to avoid conflict on the app flow

File: src/Components/Header/header.module.css [hunk 2]
```diff
@@ -3,38 +3,137 @@
...
+.background{
+  position: fixed;
+  left: 0;
+  top: 4rem;
+  width: 100vw;
+  height: 100vh;
+  background-color: rgba(0, 0, 0, 0.4);
+}
+
+.navbar{
```
Comment (cocolobos) [inline src/Components/Header/header.module.css:82]: can we add a transition when it open and close ?

File: src/Components/Header/header.module.css [hunk 3]
```diff
@@ -3,38 +3,137 @@
...
+
+.navbar a{
+  color: #000;
+  font-weight: 400;
+}
+
+.navbar h2{
+  font-size: 2rem;
+  margin-top: 40px;
+  text-decoration: underline 0.8px #ED2626;
```
Comment (cocolobos) [inline src/Components/Header/header.module.css:102]: add more weight on the underline

**Comment by cocolobos** [comment]
when you have the navBar open and scroll it causes a bu
Screen Shot 2022-06-09 at 23 36 38
g

**Review by dayanagm96 — APPROVED** [review]

**Review by gregorioelias — APPROVED** [review]

**Review by marcomigliaro — APPROVED** [review]

**Review by DeepFreq — APPROVED** [review]

**Pull request merged**